x-pack/plugins/infra/public/alerting/log_threshold/components/alert_details_app_section/components/log_rate_analysis.tsx

```diff
@@ -268,6 +268,7 @@ export const LogRateAnalysis: FC<AlertDetailsLogRateAnalysisSectionProps> = ({ r
         </EuiFlexItem>
         <EuiFlexItem>
           <LogRateAnalysisContent
+            source="observability_alerts"
```

Comment:
Maybe `observability_alert_details` or `log_threshold_alert_details` ?

Reply:
Updated in 8c6bc9da41b8dae7d4ae961a2e34b50687baf74c